src/apis/post/usePostLogin.tsx

```diff
@@ -0,0 +1,54 @@
+import { useMutation } from "@tanstack/react-query";
+import React from "react";
+import axiosInstance from "..";
+import { useNavigate } from "react-router-dom";
+
+/**
+ * 로그인하기
+ * @param  
+ *      {
+            "loginId": "string",
+            "pwd": "string"
+        }
+ * @returns 
+ */
+
+interface UserLoginData {
+  loginId: string;
+  pwd: string;
+}
+
+export const usePostLogin = () => {
+  const navigate = useNavigate();
+  const { mutate, isPending, error, isSuccess } = useMutation({
```

Comment:
```suggestion
  const { mutate:loginRequest, isPending:isLoginPending, error:loginError, isSuccess:isLoginSuccess } = useMutation({
```

위와 같은 방식으로 바로 네이밍을 할 수 있어요! 혹시 여러 데이터 fetching 기능이 한 컴포넌트에서 사용될 때에는 위와 같이 서로 다르게 네이밍해서 사용해봐도 좋을 것 같습니다~